packages/cfpb-notifications/src/molecules/notification.less

```diff
@@ -57,22 +57,29 @@
 
   // Only adding left padding if an icon is present.
   .cf-icon-svg + &_content {
-    padding-left: unit((25px / @base-font-size-px), em);
+    padding-left: unit((25px / @base-font-size-px), rem);
   }
 
   &_message {
-    .h4();
-    margin-bottom: 0;
+    font-size: unit((16px / @base-font-size-px), rem);
+    line-height: 1.25;
+    font-weight: 500;
+
+    // Tablet and above.
+    .respond-to-min(@bp-sm-min, {
+      // h4 size
+      font-size: unit((18px / @base-font-size-px), rem);
```

Comment:
Should we also add `margin-bottom: 0;` here to remove the bottom margin?

Reply:
Ah okay I noticed that the [banner notification](https://deploy-preview-1953--cfpb-design-system.netlify.app/design-system/components/banner-notification) still had bottom margin but I now realize that's because the example has a manual h4 class for some reason

<img width="861" alt="Screenshot 2024-04-12 at 1 09 50 PM" src="https://github.com/cfpb/design-system/assets/1060248/7b93fa39-a99f-4b54-a539-29950d7bb9d6">

Reply:
Ah good catch. I'll fix that

Reply:
https://github.com/cfpb/design-system/pull/1953/commits/acfe2099a5008b801020d1ba5a9d5de5bc9c859f